rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+
+var config = {
```

Comment:
u can use `const` here, this file gets transformed with babel when loaded (rollup does it internally)

Reply:
Will do. I just restored the old config from the git history.